_posts/research/2023-03-24-hierarchical-ad-hoc-agents.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "HAHA: Hierarchical Ad Hoc Agents"
+description: Using hierarchical structures to improve human-agent interaction
+author: Stéphane Aroca-Ouellette
+permalink: research/haha.html
+category: [research, learning and modeling, haha, research highlight]
+image:
+    feature: research/haha/haha_architecture.png
+    size: 50%
+excerpt_separator: <!-- More -->
+---
+
+When performing collaborative tasks with new unknown teammates, humans are particularly adept at adapting to their collaborator 
```

Comment:
Fix EOL in this abstract in a way that make sense

Reply:
This is replicated in two files, doens't make any sense:

![image](https://user-images.githubusercontent.com/4378663/228047168-2c777420-d5ac-448d-bb91-c078b9243ad8.png)